elastic/apm-server
# Enable Service Metrics by default

## Motivation/summary

In `8.5` we introduce service metrics aggregation as an experimental feature, disabled by default.
We now want to enable it by default to make it easier to use and gather feedback. 



## Checklist




## How to test these changes

0. Enable APM Server and ingest some events. 
1. Check that service metrics events are created (`metricset.name: service`).
